Dockerfile.ubuntu

```diff
@@ -60,6 +49,3 @@ RUN pip install -r /opt/file-metadata/test-requirements.txt
 
 # Install package
 RUN pip install /opt/file-metadata --upgrade
-
-# Test file-metadata import
-RUN python -c'import file_metadata; print(file_metadata.__version__)'
```

Comment:
I don't see why - Travis is already going to be testing the docker image, and all testing would happen on travis. This adds an additional commit that users would have to download